Tests/Source/Model/UserClient/UserClientKeyStoreTests.swift

```
@@ -170,17 +168,17 @@ class UserClientKeysStoreTests: OtrBaseTest {
     func testThatItMovesTheLegacyCryptobox() {
         
         // given
-        self.dynamicType.cleanOTRFolder()
+        type(of: self).cleanOTRFolder()
 
-        self.dynamicType.createFakeOTRFolder()
-        "foo".dataUsingEncoding(NSUTF8StringEncoding)!.writeToURL(UserClientKeysStore.legacyOtrDirectory.URLByAppendingPathComponent("dummy.txt"), atomically: true)
+        type(of: self).createFakeOTRFolder()
+        try? "foo".data(using: String.Encoding.utf8)!.write(to: UserClientKeysStore.legacyOtrDirectory.appendingPathComponent("dummy.txt"), options: Data.WritingOptions.atomic)
```

Should we use `try!` instead? This way it will not silently fail.